### ISSUE TITLE
[RFC] Insurance Management 

### ISSUE THREAD

**Issue by karan-dalal** [petcodeapp/dev#2]
**Summary**

Store insurance cards and information.

**Justification**

During Feb validation interviews, multiple pet owners expressed the need to manage their pets insurance. If this is going to be the one place for all their pet's info, then it would be a good idea to add insurance,

**Proposed Timeline**

Implement after March version. Not to hard, see below. More research required as to different providers and how to store info.

**Modules & Stages to implement**

Adding a button into health menu and allowing user to enter some information. We might even explore more, and offer users our own pet care insurance.

**Comment by lucaszcai** [comment]
I think that's a good idea, and definitely useful to those who want to store their insurance information somewhere. Not sure if that's something that everyone has though, and if it isn't, just make sure that it's easy to make optional

**Comment by karan-dalal** [comment]
@lucaszcai Definitely, and as I mentioned in the thing - there might be even a way to make some $$$ of this. If we contract with a company to offer users the ability to purchase the company's insurance right from our app, we could take a cut. But ofc, modularity is key.
